src/components/ingestion/FileIngestionEngine.js

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs'
+import AbstractIngestionEngine from '%src/components/ingestion/AbstractIngestionEngine'
+
+/**
+ * The FileIngestionEngine handles processing a video file. It is a concrete
+ * implementation of AbstractIngestionEngine, which reads in a file,
+ * converts it into an MPEG-TS stream represented as a series of Payloads,
+ * and then pipes that to Kafka.
+ *
+ * @param  {string}  path The path of the file to be ingested
```

Comment:
I don't know how JSDocs actually work so take this with a grain of salt, but [this example](https://jsdoc.app/howto-es2015-classes.html) seems to have constructor documentation as part of the constructor directly.

It also includes an `@extends` keyword we might want to consider here.

Reply:
Okay. Will change.